src/djangooidc/views.py

```diff
@@ -78,9 +82,13 @@ def login_callback(request):
         if user:
             login(request, user)
             logger.info("Successfully logged in user %s" % user)
+            # Double login bug?
             return redirect(request.session.get("next", "/"))
         else:
             raise o_e.BannedUser()
+    except o_e.NoStateDefined as nsd_err:
+        logger.debug(f"No State Defined: {nsd_err}")
```

Comment:
```suggestion
        logger.warning(f"No State Defined: {nsd_err}")
```
This would be good to log at a state higher than debug. I vote for warning because it is an event we can recover from easily but I'm totally fine with info or error